java/client/src/main/java/glide/api/commands/StreamBaseCommands.java

```diff
@@ -1372,4 +1372,229 @@ CompletableFuture<GlideString[]> xclaimJustId(
      */
     CompletableFuture<Map<GlideString, Object>[]> xinfoConsumers(
             GlideString key, GlideString groupName);
+
+    /**
+     * Returns information about the stream stored at key <code>key</code>.<br>
+     * To get more detailed information use {@link #xinfoStreamFull(String)} or {@link
+     * #xinfoStreamFull(String, int)}.
+     *
+     * @see <a href="https://valkey.io/commands/xinfo-stream/">valkey.io</a> for details.
+     * @param key The key of the stream.
+     * @return A <code>Map</code> of stream information for the given <code>key</code>. See the
+     *     example for a sample response.
+     * @example
+     *     <pre>{@code
+     * // example of using the API:
+     * Map<String, Object> response = client.xinfoStream("myStream").get();
+     * // the response contains data in the following format:
+     * Map<String, Object> data = Map.of(
+     *     "length", 4L,
+     *     "radix-tree-keys", 1L,
+     *     "radix-tree-nodes", 2L,
+     *     "last-generated-id", "1719877599564-0",
+     *     "max-deleted-entry-id", "0-0",
+     *     "entries-added", 4L,
+     *     "recorded-first-entry-id", "1719710679916-0",
+     *     "groups", 1L,
+     *     "first-entry", new Object[] {
+     *          new Object[] {
```

Comment:
Sorry, this object array isn't needed actually. It was only the "entries" value that needed updating. Python example:
```
b"first-entry": [
                        b"1719710679916-0",
                        [b"foo1", b"bar1", b"foo2", b"bar2"],
                    ],
```

Reply:
Oh my bad, that was copy-pasted by me originally.